packages/ui-components/src/components/pagination/index.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Icon } from '../icons';
+import { SelectBox, SelectBoxOption } from '../select-box';
+import { PaginationProps } from './types';
+import { Input } from '../input/input';
+
+export const Pagination = ({
+  rowsPerPage = [5, 10, 15, 20],
+  currentPage = 1,
+  currentRowsPerPage = 5,
+  totalPage = 1,
+  onChangeRows,
+  onChangePage,
+}: PaginationProps) => {
+  const [page, setPage] = React.useState(currentPage);
+  const [prevStyle, setPrevStyle] = React.useState('');
+  const [nextStyle, setNextStyle] = React.useState('');
+
+  const prevPage = () => {
+    const prevPage = Number(page) > 1 ? Number(page) - 1 : 1;
```

Comment:
type of the **page** is number, does not need to parse Number in the function and any function has the same.